maqs-webservices/src/test/java/com/cognizantsoftvision/maqs/webservices/WebServiceTestObjectUnitTest.java

```diff
@@ -16,14 +15,14 @@
 /**
  * The Web Service Test Object unit test class.
  */
-public class WebServiceTestObjectUnitTest extends BaseGenericTest {
+public class WebServiceTestObjectUnitTest extends BaseWebServiceTest {
```

Comment:
Suggest changing this back
These tests do not benefit from using the service base.
I would actually argue these tests are more valuable with base generic.

Reply:
changed back to baseGenericTest